client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -1,19 +1,22 @@
 import { Button, Gridicon } from '@automattic/components';
 import { translate } from 'i18n-calypso';
 import Badge from 'calypso/components/badge';
+import { isTaskDisabled } from './task-helper';
 import { Task } from './types';
 
 const ChecklistItem = ( { task }: { task: Task } ) => {
 	const { id, isCompleted, actionUrl, title } = task;
+
+	const taskDisabled = isTaskDisabled( task );
 	return (
 		<li className={ `launchpad__task-${ id }` }>
 			<Button
 				className="launchpad__checklist-item"
-				disabled={ task.isCompleted }
+				disabled={ taskDisabled }
 				href={ actionUrl }
 				data-task={ id }
 			>
-				{ task.isCompleted && (
+				{ isCompleted && taskDisabled && (
```

Comment:
Just wanted to confirm this conditional is right? That's in part because we already handle isCompleted logic in the isTaskDisabled logic, but also seems a bit odd to require both of these to be true...

Reply:
Yes, this logic is needed in order to hide the 'checkmark' icon on the left side of each checklist item. Its needed for the special case where we want to hide the icon AND disable a task.

Maybe creating a new variable that determines if the checkmark should display will make this clearer.